src/core.js

```diff
@@ -1778,9 +1791,17 @@ Strophe.Connection = class Connection {
 
         /** Variable: pass
          *  Authentication identity (User password).
+         *
          */
         this.pass = pass;
 
+        /** Variable: scramKeys
+         *  The SASL SCRAM client and server keys. This variable will be populated with a non-null
+         *  object of the above described form after a successful SCRAM connection
+         *
+         */
+        this.scramKeys = null;
```

Comment:
Strophe is a bit of a mishmash of coding styles, but the attributes on the Strophe.Connection object are all snake case.

Could you please rename `scramKeys` to `scram_keys`? Thanks.